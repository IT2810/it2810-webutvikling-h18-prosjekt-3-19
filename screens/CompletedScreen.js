import React from 'react';
import { ScrollView, StyleSheet, View, Text } from 'react-native';
import Completed from '../components/Completed';

export default class CompletedScreen extends React.Component {
  static navigationOptions = {
    title: 'Tasks Completed',
  };

  render() {
    return (
            <Completed/>
    );
  }

  _maybeRenderDevelopmentModeWarning() {

  }

  _handleLearnMorePress = () => {
    WebBrowser.openBrowserAsync('https://docs.expo.io/versions/latest/guides/development-mode');
  };

  _handleHelpPress = () => {
    WebBrowser.openBrowserAsync(
      'https://docs.expo.io/versions/latest/guides/up-and-running.html#can-t-see-your-changes'
    );
  };
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'column',
    backgroundColor: '#fff',
  },
  scrollContainer: {
    marginBottom: 100,
  },
  contentContainer: {
    paddingTop: 30,
  },
  heading: {
    fontSize: 17,
    color: 'rgba(96,100,109, 1)',
    textAlign: 'center',
    alignItems: 'center',
    marginTop: 10,
    marginBottom: 0,
  },
  headerText: {
    fontSize: 35,
  },
});