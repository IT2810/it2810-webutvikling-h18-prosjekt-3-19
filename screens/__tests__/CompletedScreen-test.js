import React from "react";
import CompletedScreen from "../CompletedScreen";
import renderer from "react-test-renderer";

test("CompletedScreen renders correctly", () => {
  const tree = renderer.create(<CompletedScreen />);
  expect(tree.toJSON()).toMatchSnapshot();
});
