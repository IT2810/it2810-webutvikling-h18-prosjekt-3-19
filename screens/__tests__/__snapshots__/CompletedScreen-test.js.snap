// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletedScreen renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Completed Stuff is here
  </Text>
</View>
`;
